### ISSUE TITLE
a11y: Identical and non-descriptive label for button "copy to clipboard"

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#14013]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

`a11y issue 116`

Identical and non-descriptive labels are specified for the button that resulted in screen reader users finding it difficult to understand the purpose of the button effectively.

API Key
Main Content
- Copy button.

**To Reproduce**

**Result**
Identical and non-descriptive labels are specified for the button that resulted in screen reader users finding it difficult to understand the purpose of the button effectively.

**Expected Result**

The screen reader should announce unique and descriptive labels for the button.

**Screenshots**

**Additional context**
